internal/gapicgen/generator/config.go

```diff
@@ -727,12 +727,20 @@ var microgenGapicConfigs = []*microgenConfig{
 		apiServiceConfigPath:  "google/monitoring/dashboard/v1/monitoring_gapic.yaml",
 		releaseLevel:          "beta",
 	},
-	{
+  {
 		inputDirectoryPath:    "google/cloud/functions/v1",
 		pkg:                   "functions",
 		importPath:            "cloud.google.com/go/functions/apiv1",
 		gRPCServiceConfigPath: "google/cloud/functions/v1/functions_grpc_service_config.json",
 		apiServiceConfigPath:  "google/cloud/functions/v1/functions_gapic.yaml",
 		releaseLevel:          "beta",
 	},
+  {
+		inputDirectoryPath:    "google/analytics/admin/v1alpha",
+		pkg:                   "admin",
+		importPath:            "cloud.google.com/go/analytics/admin/apiv1alpha",
+		gRPCServiceConfigPath: "google/analytics/admin/v1alpha/admin_grpc_service_config.json",
+		apiServiceConfigPath:  "google/analytics/admin/v1alpha/analyticsadmin_gapic.yaml",
+		releaseLevel:          "alpha",
+  },
```

Comment:
Indenting of the brackets seems off here? Surprised it's not consistent.

Reply:
Whoops, that is what I get for trying to merge on Github.